File: src/main/java/util/StringUtils.java
package util;

public class StringUtils {
    public static boolean isNullOrEmpty( String target ) {
        if( target == null || target.isEmpty() ) {
            return true;
        }
        return false;
    }

    public static boolean isNotNullOrEmpty( String target ) {
        return !isNullOrEmpty( target );
    }

    public static boolean isNull( String target ) {
        return target == null;
    }

    public static boolean isNotNull( String target ) {
        return !isNull(target);
    }
}

File: src/main/kotlin/util/StringUtils.kt
package util.kotlin

public class StringUtils {
   public fun isNullOrEmpty( target : String ) : Boolean {
      if( target == null || target.isEmpty() ) {
         return true;
      }
      return false;
   }

   fun isNotNullOrEmpty( target : String ) : Boolean {
      return !isNullOrEmpty( target );
   }

   fun isNull( target : String ) : Boolean {
      return target == null;
   }

   fun isNotNull( target : String ) : Boolean {
      return !isNull( target );
   }
}